packages/@aws-cdk/aws-ecs/lib/scheduled-ecs-task.ts

```diff
@@ -0,0 +1,108 @@
+import events = require('@aws-cdk/aws-events');
+import cdk = require('@aws-cdk/cdk');
+import { ICluster } from './cluster';
+import { ContainerImage } from './container-image';
+import { Ec2EventRuleTarget } from './ec2/ec2-event-rule-target';
+import { Ec2TaskDefinition } from './ec2/ec2-task-definition';
+import { AwsLogDriver } from './log-drivers/aws-log-driver';
+
+export interface ScheduledEc2TaskProps {
+  /**
+   * The cluster where your service will be deployed.
+   */
+  readonly cluster: ICluster;
+
+  /**
+   * The image to start.
+   */
+  readonly image: ContainerImage;
+
+  /**
+   * The schedule or rate (frequency) that determines when CloudWatch Events
+   * runs the rule. For more information, see Schedule Expression Syntax for
+   * Rules in the Amazon CloudWatch User Guide.
+   *
+   * @see http://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html
+   *
+   * You must specify this property.
+   */
+  readonly scheduleExpression: string;
+
+  /**
+   * The CMD value to pass to the container. A string with commands delimited by commas.
```

Comment:
Defaults needed on any optional props

Reply:
Updated for all except MemoryLimit and MemoryReservationLimit.